bonus-jest/demo/src/TestMe.tsx

```diff
@@ -17,21 +17,25 @@ export const TestMe = (props: TestMeProps) => {
 };
 
 export class InnerMe extends React.Component<TestMeProps, TestMeState> {
+  setState(arg0: { clicked: boolean; }): any {
+    throw new Error("Method not implemented.");
+  }
   state = {
     clicked: false
   };
 
   onClick = () => {
     this.setState({ clicked: true });
   };
+  props: any;
 
   render() {
     return !this.state.clicked ? (
       <div onClick={this.onClick} id="innerMe">
         Hello {this.props.name}, Click Me
       </div>
     ) : (
-      <div id="innerMe">Clicked</div>
-    );
+        <div id="innerMe">Clicked</div>
```

Comment:
remove this spacing

Reply:
The code editor is automatically providing spaces in this part. Is it necessary?

Reply:
Probably resulting from the ternary? In that case it is fine I overlooked the ternary

Reply:
So is it okay to let it be that way?

Reply:
> So is it okay to let it be that way?

Yes